src/main/java/nl/tudelft/remla/app/controllers/HomeController.java

```diff
@@ -0,0 +1,15 @@
+package nl.tudelft.remla.app.controllers;
```

Comment:
Don't think this file is needed, the GET mapping for `/` is in the `SentimentController`